Jenkinsfile

```diff
@@ -628,7 +630,9 @@ def dotnet(Closure body){
     ./dotnet-install.sh --install-dir "\${DOTNET_ROOT}" -version '6.0.100'
     """)
     withAzureCredentials(path: "${homePath}", credentialsFile: '.credentials.json') {
-      withTerraform(){
+      withTerraformEnv(version: '0.15.3'){
+        sh 'which terraform || true'
+        sh 'terraform --version || true'
```

Comment:
The installation seems to work

<img width="1323" alt="image" src="https://user-images.githubusercontent.com/2871786/169061531-7682f2d9-885c-43e5-b14f-7682135fd7ae.png">

```suggestion
        sh 'which terraform || true'
        sh 'terraform --version || true'
```